lib/new_relic/agent/external.rb

```diff
@@ -23,7 +23,7 @@ module External
       NON_HTTP_CAT_SYNTHETICS_HEADER = 'NewRelicSynthetics'.freeze
       NON_HTTP_CAT_CONTENT_LENGTH = -1
 
-      # Process obfuscated +String+ indentifying a calling application and transaction that is also running a
+      # Process obfuscated +String+ identifying a calling application and transaction that is also running a
```

Comment:
`indent` vs `ident`. I'm hoping the tooling picked the right one

Reply:
Seems it did!